### PR TITLE
Change lister user to root and improve documentation

### PR DESCRIPTION
**Pull request by nanndoj**
Hi guys,
Great job on this project!! 

As an outside developer, I found some small things I need to fix in order to be able to put it to run smoothly:
 - Had to change the default user for the lister Dockerfile from jenkins to **root**
 - Need to dig into the documentation to find how to build the RPM for the collector module
 - There's no information on how to build the RMP package for the lister module

### CODE REVIEW

File: xroad-catalog-collector/README.md [hunk 1]
```diff
@@ -32,6 +32,7 @@ Then run the collector with profile sshtest
 First make sure that xroad-catalog-persistence is located next to xroad-catalog-collector. The RPM build
  uses sql files from xroad-catalog-persistence/src/main/sql.
  
-    $ gradle clean build
+    $ ./gradlew clean build
```
Comment (nanndoj) [inline xroad-catalog-collector/README.md]: I was receiving this error when trying to build without using the wrapper

```
FAILURE: Build failed with an exception.

* Where:
Build file '/Users/fernandosantos/Projects/xroad-catalog/build.gradle' line: 33

* What went wrong:
A problem occurred evaluating root project 'xroad-catalog'.
> Cannot add task 'wrapper' as a task with that name already exists.

```

File: xroad-catalog-lister/packages/xroad-catalog-lister/docker/Dockerfile [hunk 2]
```diff
@@ -5,5 +5,5 @@ RUN yum -y install sudo git rpm-build java-1.8.0-openjdk-headless
 RUN yum clean all
 RUN sed -i 's/requiretty/!requiretty/' /etc/sudoers
 
-USER jenkins
+USER root
```
Comment (nanndoj) [inline xroad-catalog-lister/packages/xroad-catalog-lister/docker/Dockerfile]: The user jenkis may not be present.

**Comment by jansu76** [comment]
Thank you for the PR @nanndoj ! I ended up doing some additional updates to documentation to use gradlew consistently and modified the instructions on docker rpm builds further. How does https://github.com/vrk-kpa/xroad-catalog/tree/PR-7-change-user-root look to you?

User change jenkins -> root has an effect on the built packages ownership, so we have to test our build pipeline before this change can be merged to master.

**Comment by jansu76** [comment]
Changes implemented in a different PR https://github.com/vrk-kpa/xroad-catalog/pull/8 instead

**Comment by nanndoj** [comment]
Hi @jansu76,

Thanks for the changes. I reviewed https://github.com/vrk-kpa/xroad-catalog/tree/PR-7-change-user-root and it looks great. Awesome job!

**Pull request closed without merging**